package.json

```
@@ -41,7 +41,7 @@
     "eslint-plugin-jsdoc": "^46.0.0",
     "eslint-plugin-local": "link:./.eslintplugin",
     "eslint-plugin-markdown": "^3.0.0",
-    "eslint-plugin-prettier": "^4.0.0",
+    "eslint-plugin-prettier": "^5.0.0-alpha.2",
```

Now v5.0.0 is available

```suggestion
    "eslint-plugin-prettier": "^5.0.0",
```